### DISCUSSION TITLE
Get your spaCy stickers!

### DISCUSSION THREAD

**Discussion by ines** [explosion/spaCy#6568] [News & Announcements]
With 2020 coming to an end and spaCy v3 just around the corner, we hope we can brighten up your 2021 with some stickers! Including all new designs and ✨holographic✨ styles. It's FREE and we ship worldwide.

💌 **Get yours here:** https://docs.google.com/forms/d/e/1FAIpQLSdqh1DRJWT7S3dxDYo86fLlvet3i0jqXPaHk7InlYnC1mAuBA/viewform

![stickers_2020](https://user-images.githubusercontent.com/13643239/102150955-d2243600-3ec5-11eb-8638-5f75671d2e6a.jpg)

**Comment by AmeliaMN** [comment]
Hello! Are you still sending stickers occasionally? I realized that this announcement is pretty old, but I am teaching spaCy this semester, and I had filled out the google form a few months ago hoping to get a few. If this is no longer the way to get stickers, can you point me in the right direction?

**Comment by eigenvektorin** [comment]
Since I never got mine when I filled out the forms in 2020 ( :-( ) I also wanted to ask the same thing.

**Comment by goonhoon** [comment]
Since this thread got revived I will hop in to say that I would potentially be interested to even buy some to support SpaCy and get some cool stickers in return.

**Comment by adrianeboyd** [comment]
We don't have a stickers form open, but for right now we can still send stickers occasionally. You can contact us at contact@explosion.ai.

(Note that we can't make any promises about doing this further in the future and this comment might get deleted!)
